addons/sale_project/tests/test_project_profitability.py

```diff
@@ -234,13 +240,80 @@ def test_project_profitability(self):
                 },
             },
         )
+        # Create a down payment for a fixed amount of 115
+        context = {
+            'active_model': 'sale.order',
+            'active_ids': [self.sale_order.id],
+            'active_id': self.sale_order.id,
+            'default_journal_id': self.company_data['default_journal_sale'].id,
+        }
```

Comment:
```suggestion
        Downpayment = self.env['sale.advance.payment.inv'].with_context({
            'active_model': 'sale.order',
            'active_ids': self.sale_order.ids,
            'active_id': self.sale_order.id,
            'default_journal_id': self.company_data['default_journal_sale'].id,
        })
```
This is also possible, and slightly preferable imho :slightly_smiling_face: